tests/test_interruptions.py

```diff
@@ -104,6 +125,12 @@ def session_interrupt():
 
 
 @pytest.fixture
+def test_interrupt():
```

Comment:
Please call this test_interrupt_callback or test_interrupt_hook. It is confusing along with the `test_interrupt` hook...